autolift-scalaz/src/main/scala/autolift/scalaz/LiftAp.scala

```diff
@@ -44,7 +44,7 @@ final class LiftedAp[A, B, F[_]](protected val f: F[A => B])(implicit ap: Apply[
 }
 
 trait LiftedApImplicits{
-	implicit def liftedApFunctor[A, F[_]] = new Functor[LiftedAp[A, ?, F]]{
+	implicit def liftedApFunctor[A, F[_]] = new _root_.scalaz.Functor[LiftedAp[A, ?, F]]{
```

Comment:
Because the names Functor and scalaz are already taken locally autolift.scalaz.Functor. Has nothing to do with macros just name clash. Could also be resolved by named imports.